### PR TITLE
[WIP] bspwm module 

### PR DESCRIPTION
**Pull request by NefixEstrada**
[DO NOT MERGE YET]

This is the initial commit for the bspwm module. Right now, you can add `bspc config` directives, startup programs and raw configuration.

Based in this issue: #359

### CODE REVIEW

File: modules/services/window-managers/bspwm.nix [hunk 1]
```diff
@@ -0,0 +1,138 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+	cfg = config.xsession.windowManager.bspwm;
```
Comment (rycee) [inline modules/services/window-managers/bspwm.nix]: Please make sure to use two spaces per indentation level, which is sort-of standard within the Nix community.

Comment (NefixEstrada) [reply modules/services/window-managers/bspwm.nix]: Okay, I'll do that in the following commit

**Comment by babariviere** [comment]
Hello, what's the advancement for this module ? Do you need some help ?

**Comment by NefixEstrada** [comment]
You can check the status of the PR at #359 . I stopped working on this a year ago, and I don't use bspwm now. Feel free to finish it!

**Comment by rycee** [comment]
Closed by #981

**Pull request closed without merging**